modules/manage/pages/cluster-maintenance/nodewise-partition-recovery.adoc

```diff
@@ -1,28 +1,36 @@
+Launch node-wise partition recovery with `rpk cluster partitions unsafe-recover`. This command includes an interactive prompt to confirm execution of the generated recovery plan, as it is a destructive operation. When you trigger node-wise partition recovery, the partitions on the node are rebuilt on a best-effort basis. As a result of executing this operation, you may lose some data that has not yet been replicated to the surviving partition replicas.
 
 The syntax for this command is as follows:
 
  rpk cluster partitions unsafe-recover --from-nodes 1,3,5
 
-The `--from-nodes` parameter accepts a comma-delineated list of dead node IDs you wish to recover the data from. The above example would perform recovery operations on nodes 1, 3, and 5. Redpanda will assess these nodes to identify which partitions lack majority. It will then create a plan to recover the impacted partitions and prompt you to confirm it. You must respond `yes` to continue with recovery.
+The `--from-nodes` flag accepts a comma-separated list of the dead node IDs you wish to recover the data from. This example performs recovery operations on nodes 1, 3, and 5. Redpanda assesses these nodes to identify which partitions lack majority. It then creates a plan to recover the impacted partitions and prompts you for confirmation. You must respond `yes` to continue with recovery.
+
+The `--dry` flag performs a dry run and allows you to view the recovery plan with no risk to your cluster.
+
+[NOTE] 
+====
+When running node-wise partition recovery, it's possible that there may be more recent data (a higher offset) available in Tiered Storage if:
+
+* Raft replication was stuck or slow before the node failure
+* Zero live replicas remain in the cluster (because the partition had a replication factor of one, `RF=1`)
 
-You may also optionally add the `--dry` parameter to this command. This will perform a dry run and allow viewing the recovery plan with no risk to your cluster.
+If partition data is available in the object store, it is still available for consumers to read. Only data stored locally but not uploaded to the object store would be lost.
+====
 
-Once the recovery operation is started, you may monitor the status of its execution using the `rpk cluster partitions balancer-status` command. The recovery operation can take some time to complete, especially when a lot of data is involved. This command allows you to monitor progress in real-time.
+After the recovery operation is started, you can monitor the status in real-time with `rpk cluster partitions balancer-status`. The recovery operation can take some time to complete, especially when a large amount of data is involved.
```

Comment:
Ideally we'd show an example output when recovery is happening

Reply:
Did you look at https://deploy-preview-842--redpanda-docs-preview.netlify.app/24.3/manage/cluster-maintenance/nodewise-partition-recovery/#example-recovery-operation?